core-android/src/main/java/com/uber/sdk/android/core/utils/AppProtocol.java

```diff
@@ -26,20 +26,19 @@
 import static com.uber.sdk.android.core.SupportedAppType.UBER_EATS;
 
 public class AppProtocol {
-    @Deprecated
```

Comment:
This was unused. Should it not be removed?

Reply:
It's `public` so it can't be deleted without a major version upgrade.  Changes to visibility are slated for 1.0

Reply:
Will restore.